components/tooltip/tooltip.test.js

```diff
@@ -166,27 +166,26 @@ describe('DtTooltip tests', function () {
         });
       });
 
-      describe('When focusin tooltip', function () {
-        beforeEach(async function () {
-          // set data-focus-visible-added attribute to emulate visible focus
-          wrapper.vm.$refs.anchor.firstElementChild.setAttribute('data-focus-visible-added', '');
-          await anchor.trigger('focusin');
-        });
-
-        it('should display tooltip', function () {
-          assert.isTrue(tooltip.isVisible());
-        });
-
-        describe('When escape is pressed', function () {
-          beforeEach(async function () {
-            await anchor.trigger('keydown', { key: 'Escape' });
-          });
-
-          it('should hide tooltip', function () {
-            assert.isFalse(tooltip.isVisible());
-          });
-        });
-      });
+      // jsdom does not yet support :focus-visible, commenting this out for now...
+      // describe('When focusin tooltip', function () {
+      //   beforeEach(async function () {
+      //     await anchor.trigger('focusin');
+      //   });
+
+      //   it('should display tooltip', function () {
+      //     assert.isTrue(tooltip.isVisible());
+      //   });
+
+      //   describe('When escape is pressed', function () {
+      //     beforeEach(async function () {
+      //       await anchor.trigger('keydown', { key: 'Escape' });
+      //     });
+
+      //     it('should hide tooltip', function () {
+      //       assert.isFalse(tooltip.isVisible());
+      //     });
+      //   });
+      // });
```

Comment:
Tried a couple options and nothing seems to work, it's OK to comment this for now

Reply:
Yeah I know haha, really annoying. There is an open issue for it https://github.com/jsdom/jsdom/issues/3055